subset/security/security_passwords/src/main/test_docker_files/Dockerfile.faux_fail

```diff
@@ -0,0 +1,31 @@
+# Dockerfile for running the corresponding faux device.
```

Comment:
What's this file for?  This doesn't seem right...

Reply:
The files in test_docker_files/ folder were local tests I made to understand the code logic, and also to test my changes a bit quicker. They actually serve no purpose other than testing the code and can be removed with no effect. I just left it in as it might help others working on this test in the near future. 

Thanks!